src/unix/async.c

```diff
@@ -33,6 +33,9 @@
 #include <string.h>
 #include <unistd.h>
 
+#define ASYNC_PENDING (1<<0)
+#define ASYNC_BUSY    (1<<1)
```

Comment:
Can you name these `UV__ASYNC_BUSY` (to avoid symbol conflicts) and write the expression as `(1 << 1)`?